pom.xml

```diff
@@ -553,6 +553,40 @@
       </build>
     </profile>
 
+    <profile>
+      <id>build-eclipse-2020-03</id>
+      <activation>
+        <property>
+          <name>eclipse.target</name> <value>2020-03</value>
+        </property>
+      </activation>
+      <properties>
```

Comment:
QQ: what are these, and until when should we carry them?

Reply:
This is the selection mechanism for 2020-03 (vs the default, which is still 2018-09.

But maybe you mean the jetty properties?  I think we carried them over from GPE.  We could get rid of them since I think all Eclipse releases we support use Jetty 9.4.x.

Reply:
Ah, yeah, was talkinf about the jetty properties.